lib/engine.rb

```diff
@@ -0,0 +1,48 @@
+class Engine
+  def initialize(engine)
+    @engine = engine
+  end
+
+  def bamboo_xml
+    @bamboo_xml ||= <<~XML
+      <enginelist>
+        #{engine_xml}
+      </enginelist>
+    XML
+  end
+
+  def engine_xml
+    @engine_xml ||= <<~XML
+      <engine code="#{engine}">
+        <supportsLanguageCode>grc</supportsLanguageCode>
+        <supportsLanguageCode>la</supportsLanguageCode>
+        <supportsLanguageCode>lat</supportsLanguageCode>
+        <supportsOption>strictCase=^1$</supportsOption>
```

Comment:
The Perseids version of Morphsvc has the options `strictCase` and `checkPreverbs`. I removed `checkPreverbs` because it doesn't seem to do anything. The functionality is [commented out](https://github.com/perseids-tools/morpheus-perseids/blob/master/src/anal/checkpreverb.c#L20) in the Morpheus code.

The Alpheios version supports `noAposRetry` ([code](https://github.com/alpheios-project/morphsvc/blob/master/morphsvc/lib/engines/MorpheusLocalEngine.py#L129)). @balmas do you think it's worth adding this functionality? Was there a particular word or set of words that caused you to add this to the Alpheios version?

Reply:
This is the issue from Alpheios that led to this change: https://github.com/alpheios-project/data-models/issues/61

The need (or not) for it depends upon how sensitive you want to be to tokenization and encoding errors. Alpheios tries to work with any Greek text on the web, which means we have to account for various possibilities that might not be so important for Perseids.

Reply:
Thanks, that makes sense.